### PR TITLE
add more lxqt files

### PR DESCRIPTION
**Pull request by Besanon**

### CODE REVIEW

File: apparmor.d/groups/lxqt/lxqt-globalkeysd [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+@{exec_path} = @{bin}/lxqt-globalkeysd
+profile lxqt-globalkeysd @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/bus-accessibility>
+  include <abstractions/video>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/gvfs-open>
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-globalkeysd:18]: Don't use `gvfs-open` directly. You need someting like: `@{open_path} rPx -> child-open,` here.

**Comment by roddhjav** [comment]
Most basic UI app can open external resource to show the help page. Therefore you need something like the following in most of these profiles:
```
  @{open_path} rPx -> child-open-help,
```

Have a look at the xfce app as example.

File: apparmor.d/groups/lxqt/lxqt-globalkeysd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  @{exec_path} mr,
+
+  @{bin}/screengrab rpx,
+  @{bin}/lxqt-config-brightness rpx,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-globalkeysd:23]: `rPx` instead of `rpx` (unless it is intentional)

**Comment by barmogund** [comment]
altered the abstractions in the other profiles as well, seems to be ok

File: apparmor.d/groups/lxqt/lxqt-globalkeysd (hunk 1 above)
Comment (Besanon) [reply apparmor.d/groups/lxqt/lxqt-globalkeysd]: ok, its used in lxqt-globalkeysd. I altered it in 2 other occurences in profiles to be considered later

File: apparmor.d/groups/lxqt/lxqt-about [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-about
+profile lxqt-about @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/lxqt>
+  include <abstractions/video>
```
[the remark this answers is not in the document]
Comment (Besanon) [reply apparmor.d/groups/lxqt/lxqt-about]: Its not needed, a left-over from an old version. I will check several other profiles in the group whether graphics or video is used

**Comment by roddhjav** [comment]
Sorry for the delay, it looks good now. Merged!

**Pull request merged**